src/services/googleSheetsService.ts

```
@@ -0,0 +1,259 @@
+            this.gisInited = true;
+            resolve();
+          };
+          document.head.appendChild(gsiScript);
+        });
+      } else {
+        this.gisInited = true;
+      }
+
+      resolve();
+    });
+  }
```

Biome (2.1.2)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)